modules/tensor_mechanics/include/materials/ADLAROMANCEStressUpdateBase.h

```diff
@@ -101,9 +116,30 @@ class ADLAROMCreepStressUpdate : public ADRadialReturnCreepStressUpdateBase<comp
   ADReal
   computePolynomial(const ADReal & value, const unsigned int degree, const bool derivative = false);
 
-private:
-  /// Userobject that holds ROM data set
-  const LAROMData & _rom;
+  /// Calculates and returns the transformed limits for the ROM calculations
+  std::vector<std::vector<std::vector<Real>>> getTransformedLimits() const;
+
+  /// Calculates and returns vector utilized in assign values
+  std::vector<std::vector<unsigned int>> getMakeFrameHelper() const;
+
+  /// Returns vector of the functions to use for the conversion of input variables.
+  virtual std::vector<std::vector<ROMInputTransform>> getTransform();
```

Comment:
Make this `virtual std::vector<std::vector<ROMInputTransform>> getTransform() = 0` to require the derived class to implement it.  Then get rid of the implementation in this class that just throws an error.  Same deal for several of the other methods in this class as well.

Reply:
Oh yeah, that makes sense! I had it that way for a reason, but I can't remember why...